sklearn/linear_model/stochastic_gradient.py

```diff
@@ -667,7 +667,12 @@ def _fit_multiclass(self, X, y, alpha, C, learning_rate,
                 self.intercept_ = self.standard_intercept_
 
     def partial_fit(self, X, y, classes=None, sample_weight=None):
-        """Fit linear model with Stochastic Gradient Descent.
+        """Carry out stochastic gradient descent on one pass of X, y.
```

Comment:
Let's rephrase this a little:
"Perform one epoch of stochastic gradient descent on the given samples"
WDYT?

Reply:
Sounds good.